core/pom.xml

```diff
@@ -680,5 +680,75 @@
 				</dependency>
 			</dependencies>
 		</profile>
+		<profile>
+			<id>jdk11</id>
+			<activation>
+				<jdk>11</jdk>
+			</activation>
+			<dependencies>
+				<dependency>
+					<groupId>cglib</groupId>
+					<artifactId>cglib-nodep</artifactId>
+					<version>3.2.7</version>
+				</dependency>
+				<dependency>
+					<groupId>org.apache.cxf</groupId>
+					<artifactId>cxf-rt-frontend-jaxws</artifactId>
+					<version>${cxf.version}</version>
+					<exclusions>
+						<exclusion>
+							<groupId>com.sun.xml.messaging.saaj</groupId>
+							<artifactId>saaj-impl</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.apache.geronimo.specs</groupId>
+							<artifactId>geronimo-jta_1.1_spec</artifactId>
+						</exclusion>
+					</exclusions>
+				</dependency>
+				<dependency>
+					<groupId>org.apache.cxf</groupId>
+					<artifactId>cxf-rt-transports-http</artifactId>
+					<version>${cxf.version}</version>
+					<exclusions>
+						<exclusion>
+							<groupId>com.sun.xml.messaging.saaj</groupId>
+							<artifactId>saaj-impl</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.apache.geronimo.specs</groupId>
+							<artifactId>geronimo-jta_1.1_spec</artifactId>
+						</exclusion>
+					</exclusions>
+				</dependency>
+				<dependency>
+					<groupId>org.apache.cxf</groupId>
+					<artifactId>cxf-rt-rs-client</artifactId>
+					<version>${cxf.version}</version>
+					<exclusions>
+						<exclusion>
+							<groupId>com.sun.xml.messaging.saaj</groupId>
+							<artifactId>saaj-impl</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.apache.geronimo.specs</groupId>
+							<artifactId>geronimo-jta_1.1_spec</artifactId>
+						</exclusion>
```

Comment:
Is it possible to exclude these by default? Do we need the JTA spec compile time?